### PR TITLE
ffuf: add page

### PR DESCRIPTION
**Pull request by p3rception**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/ffuf.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- Enumerate subdomains by changing the position of the keyword:
+
+`ffuf -w {{path/to/subdomains.txt}} -u {{http://FUZZ.target.com}}`
+
+- Fuzz with specified [t]hreads (default: 40) and pro[x]ying the traffic. Also save [o]utput to a file:
+
+`ffuf -o -w {{path/to/wordlist.txt}} -u {{http://target/FUZZ}} -t {{500}} -x {{http://127.0.0.1:8080}}`
+
+- Fuzz specific Header ("Name: Value") and match HTTP responses:
```
Comment (gutjuri) [inline pages/common/ffuf.md:19]:
```suggestion
- Fuzz a specific header ("Name: Value") and match HTTP responses:
```

File: pages/common/ffuf.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`ffuf -w {{path/to/wordlist.txt}} -u {{http://target.com}} -H "{{Host: FUZZ}}" -mc {{200}}`
+
+- Fuzz with specified HTTP method and payloa[d], while filtering out comma seperated status codes:
```
Comment (gutjuri) [inline pages/common/ffuf.md:23]:
```suggestion
- Fuzz with specified HTTP method and payloa[d], while filtering out comma separated status codes:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some minor suggestions.

File: pages/common/ffuf.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# ffuf
+
+> A fast web fuzzer written in Go.
+> The `“FUZZ”` keyword is used as a placeholder. Ffuf will try to hit the URL by replacing the word `“FUZZ”` with every word in the wordlist.
```
Comment (kbdharun) [inline pages/common/ffuf.md:4]:
```suggestion
> The `“FUZZ”` keyword is used as a placeholder. `ffuf` will try to hit the URL by replacing the word `“FUZZ”` with every word in the wordlist.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by gutjuri — APPROVED** [review]
Good!

File: pages/common/ffuf.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Enumerate subdomains by changing the position of the keyword:
+
+`ffuf -w {{path/to/subdomains.txt}} -u {{http://FUZZ.target.com}}`
+
+- Fuzz with specified [t]hreads (default: 40) and pro[x]ying the traffic and save [o]utput to a file:
+
+`ffuf -o -w {{path/to/wordlist.txt}} -u {{http://target/FUZZ}} -t {{500}} -x {{http://127.0.0.1:8080}}`
+
+- Fuzz a specific header ("Name: Value") and match HTTP responses:
```
Comment (acuteenvy) [inline pages/common/ffuf.md:19]:
```suggestion
- Fuzz a specific header (`Name: Value`) and match HTTP responses:
```

File: pages/common/ffuf.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# ffuf
+
+> A fast web fuzzer written in Go.
+> The `“FUZZ”` keyword is used as a placeholder. `ffuf` will try to hit the URL by replacing the word `“FUZZ”` with every word in the wordlist.
```
Comment (acuteenvy) [inline pages/common/ffuf.md:4]:
```suggestion
> The `FUZZ` keyword is used as a placeholder. `ffuf` will try to hit the URL by replacing the word `FUZZ` with every word in the wordlist.
```

File: pages/common/ffuf.md (hunk 4 above)
Comment (p3rception) [reply pages/common/ffuf.md:19]: Not sure if this change is necessary.

**Review by vitorhcl — APPROVED** [review]
Thanks for your contribution! I suggested some more mnemonics.

File: pages/common/ffuf.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# ffuf
+
+> A fast web fuzzer written in Go.
+> The `FUZZ` keyword is used as a placeholder. `ffuf` will try to hit the URL by replacing the word `FUZZ` with every word in the wordlist.
+> More information: <https://github.com/ffuf/ffuf>.
```
Comment (vitorhcl) [inline pages/common/ffuf.md:5]:
```suggestion
> More information: <https://github.com/ffuf/ffuf#usage>.
```
What do you think of directing to the usage section directly?

File: pages/common/ffuf.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+`ffuf -w {{path/to/subdomains.txt}} -u {{http://FUZZ.target.com}}`
+
+- Fuzz with specified [t]hreads (default: 40) and pro[x]ying the traffic and save [o]utput to a file:
+
+`ffuf -o -w {{path/to/wordlist.txt}} -u {{http://target/FUZZ}} -t {{500}} -x {{http://127.0.0.1:8080}}`
+
+- Fuzz a specific header ("Name: Value") and match HTTP responses:
+
+`ffuf -w {{path/to/wordlist.txt}} -u {{http://target.com}} -H "{{Host: FUZZ}}" -mc {{200}}`
```
[the remark this answers is not in the document]
Comment (p3rception) [reply pages/common/ffuf.md:21]: Is there a difference if it's [h]eader or [H]eader?

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/ffuf.md:21]: > Is there a difference if it's [h]eader or [H]eader?

We generally use the same case as the short option, but anything works.

**Comment by p3rception** [comment]
Needs anything else for merging?

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**